client/python/README.md

```diff
@@ -1,19 +1,73 @@
-# Armada Python client
+Armada Python client
+=
+
+# Installation:
+
+On Linux Machine
+=
+1) Open your favorite shell, for example, good old [Bourne Again SHell, aka, bash](https://www.gnu.org/software/bash/), the somewhat newer [Z shell, aka, zsh](https://www.zsh.org/), or shiny new [fish](https://fishshell.com/).
+2) Install [Git](https://git-scm.com/) by running `sudo apt-get install git-all` on [Debian-based](https://www.debian.org/) distributions like [Ubuntu](https://ubuntu.com/), or sudo dnf install git on [Fedora](https://getfedora.org/) 
```

Comment:
Do you need single-back-quotes around the "sudo dnf...git" command?

Reply:
The more specifics we put into this document, the more likely it is to degrade over time.

Do we really need to detail how to install `git`? If someone is already at this README, they have some concept of git repos existing, and likely can find other howtos for this level of information.